t/mojo/promise.t

```diff
@@ -349,7 +349,7 @@ subtest 'All settled (with rejection)' => sub {
   is_deeply \@results, $result, 'promise resolved';
 };
 
-subtest 'Settle with promise' => sub {
+subtest 'Settle with settled promise' => sub {
```

Comment:
These tests are unrelated to your PR, why would you randomly change their description?

Reply:
I was under the impression that a patch needs to leave the repo in a consistent state. It would seem strange to have a subtest that would test "Settle with promise", and another one that would test "Settle with unsettled promise". Because, what would the second subtest's reason for existence be, if its case is already being tested by the first subtest? So I did this modification.

Shall I revert that line?